app/web_hub_test.go

```diff
@@ -100,6 +100,6 @@ func TestHubStopRaceCondition(t *testing.T) {
 	select {
 	case <-done:
 	case <-time.After(15 * time.Second):
-		t.Fatalf("hub call did not return within 15 seconds after stop")
+		require.False(t, true, "hub call did not return within 15 seconds after stop")
```

Comment:
I missed the whole point of `require.False(t,true)` in my first review. We can just use `require.FailNow` here.

Reply:
done